src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -760,7 +760,7 @@ void LSHSearch<SortPolicy, MatType>::ReturnIndicesFromTable(
       // Map each probing bin to a bin in secondHashTable (just like we did for
       // the primary hash table).
       hashMat(arma::span(1, T), i) = // Compute code of rows 1:end of column i
-        arma::conv_to< arma::Col<size_t> >:: // floor by typecasting to size_t
+        ConvTo< arma::Col<size_t> >:: // floor by typecasting to size_t
```

Comment:
```suggestion
        ConvTo<arma::Col<size_t>>:: // floor by typecasting to size_t
```
I think the extra spaces were here because it was pre-C++11...